### ISSUE TITLE
Updated design for new grid card items

### ISSUE THREAD

**Issue by jacksonh** [omnivore-app/omnivore#1097]
We should probably create a new class and create these new grid cards in there instead of trying to use the existing ones from the home feed grid.

<img width="413" alt="Screen Shot 2022-08-18 at 19 26 40" src="https://user-images.githubusercontent.com/75189/185384170-3a4497eb-631c-4506-b9a3-79fe35cba239.png">

**Comment by jacksonh** [comment]
@rupin90 we probably need to start more about this one before we start, I just wanted to create the ticket.

**Comment by rupin90** [comment]
- How many labels are allowed to display per card?  Is there a limit to how many labels are displayed ? What if labels are so many that we need to display it in 3 to 4 lines? What's the limit?

**Comment by rupin90** [comment]
Are you saying I should just go ahead with creating a new Card component?

**Comment by rupin90** [comment]
It seems like this is what we have right now and this task can be closed? @jacksonh

**Comment by rupin90** [comment]
Fixed the bug and it iss merged so closing this one out now.
